### PR TITLE
Removed Commented code lines

### LINKED ISSUE #80: Remove the commented-out code snippets in the TrackProgressScreen.js
**Is your feature request related to a problem? Please describe.**

- Currently, There are commented-out code snippets in some components.

**Describe the solution you'd like**

1. Go to `src\pages\TrackProgressScreen.js`
2. Remove the code lines that are commented
3. Sent a pull request with the relevant changes

### PR DESCRIPTION
**Pull request by hparihar-07**
Fix #80 by removing all the commented code lines

### CODE REVIEW

**Comment by anjula-sack** [comment]
@hparihar-07 Please always make sure an issue is assigned for you before making a pull request. Don't make PRs for the issues that you are not assigned. Always drop a comment on the issue mentioning that you are interested at least.

**Comment by hparihar-07** [comment]
Sorry for that I found that no one was replying

**Comment by anjula-sack** [comment]
> Sorry for that I found that no one was replying

No worries :)

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Remove this console log as well,

https://github.com/tillioss/tilli-web-app/blob/3a3401ac6ce8902d8466dba46322a219bb3bd99f/src/pages/TrackProgressScreen.js#L81

**Comment by hparihar-07** [comment]
> Remove this console log as well,
> 
> https://github.com/tillioss/tilli-web-app/blob/3a3401ac6ce8902d8466dba46322a219bb3bd99f/src/pages/TrackProgressScreen.js#L81

Ok sir

File: src/pages/TrackProgressScreen.js [hunk 1]
```diff
@@ -77,8 +77,8 @@ class TrackProgressScreen extends Component {
 
         let responseData = await doConnect("getfeedbackCaptureList", "POST", postJson);
         if (responseData) {
-            // console.log("ordered",loopLikedCount - Object.keys(responseData.response.liked).length)
-            console.log(responseData)
+            
+
```
Comment (anjula-sack) [inline src/pages/TrackProgressScreen.js:81]:
```suggestion           
```

File: src/pages/TrackProgressScreen.js [hunk 2]
```diff
@@ -126,16 +126,16 @@ class TrackProgressScreen extends Component {
             <React.Fragment>
                 <div className='trackProgBody'>
                     <div className='emotionChat'>
-                        {/* <--------------Header----------------> */}
-                        <div className='headers mt-2'>
+
```
Comment (anjula-sack) [inline src/pages/TrackProgressScreen.js:129]:
```suggestion
```

File: src/pages/TrackProgressScreen.js [hunk 3]
```diff
@@ -126,16 +126,16 @@ class TrackProgressScreen extends Component {
...
+                       <div className='headers mt-2'>
                             <div className='d-flex justify-content-start backbtn ml-3' style={{ width: '10%' }}>
                                 <img src={Back_Button} style={{ width: '50px' }} onClick={() => { this.props.history.push('/' + MyConstant.keyList.projectUrl + '/newparentsscreen') }} alt=""/>
                             </div>
                             <div className='d-flex justify-content-center mt-2 mx-4 headtext' style={{ width: '68%' }}>
                                 <h3 className='headerStr'>Track Progress</h3>
                             </div>
                         </div>
-                        {/* <--------------Header----------------> */}
+
```
Comment (anjula-sack) [inline src/pages/TrackProgressScreen.js:138]:
```suggestion
```

**Review by anjula-sack — APPROVED** [review]
Please remove the mentioned empty lines as well

**Pull request merged**